ci/jenkins/test-vm.sh

```diff
@@ -48,6 +43,17 @@ Run K8s e2e community tests (Conformance & Network Policy) or Antrea e2e tests o
         --workdir                Home path for Go, vSphere information and antrea_logs during cluster setup. Default is $WORKDIR.
         --registry               The docker registry to use instead of dockerhub."
 
+# VM configuration
+declare -A LINUX_HOSTS_TO_IP
+declare -A WINDOWS_HOSTS_TO_IP
+declare -a LIN_HOSTNAMES=("vmbmtest0-1")
+declare -a WIN_HOSTNAMES=("vmbmtest0-win-0")
```

Comment:
ditto

Reply:
Also this PR is only updating the existing CI to use install script (instead of running these commands on the CI script).